float64 stamp 	# time of measurement
uint32 id 		# serial number of message

pid_controller[] controllers